doc/en/user/source/services/wms/vendor.rst

```diff
@@ -15,14 +15,14 @@ GetCapabilities Request
 format
 ^^^^^^
 
-The ``format`` parameter can be used to request capabilities documents in a certain format. If the requested format is not supported the default format will be used.
+The ``format`` parameter can be used to request capabilities documents in a certain format. If the requested format is not supported the default format of ``application/vnd.ogc.wms_xml`` will be used.
```

Comment:
the WMS 1.3.0 only supports `text/xml` afaik see eg. https://snapshot.tailormap.nl/geoserver/ows?service=WMS&version=1.3.0&request=GetCapabilities

Reply:
Yes, but WMS 1.1.0 uses the cited format instead:

```
curl "https://snapshot.tailormap.nl/geoserver/ows?service=WMS&version=1.1.0&request=GetCapabilities" -D - -o /dev/null
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0HTTP/2 200 
cache-control: max-age=0, must-revalidate
content-disposition: inline; filename=getcapabilities_1.3.0.xml
content-type: application/vnd.ogc.wms_xml
date: Mon, 08 Jan 2024 10:13:41 GMT
x-content-type-options: nosniff
x-frame-options: SAMEORIGIN
```

In the end, it depends on the WMS version being used.

Reply:
Thanks for the feedback, I have made it more generic